package.json

```diff
@@ -12,14 +12,17 @@
   },
   "homepage": "https://thelounge.github.io/",
   "scripts": {
-    "coverage": "nyc mocha",
+    "coverage": "rm -rf .nyc_output/ && npm-run-all coverage:* && cp test/.nycrc-report .nycrc && nyc report json && rm .nycrc",
```

Comment:
The way this works, I tell `nyc` to not clean up temporary JSON files, empty the folder when starting up, and at the end build a report combining both. In the future, we might want to skip this step, and just maintain 2 reports. I'm debating doing it now or not.